src/dataUriTextDocumentContentProvider.ts

```diff
@@ -1,7 +1,12 @@
 'use strict';
 import * as vscode from 'vscode';
 import * as Url from 'url';
+import * as path from 'path';
 import * as fs from 'fs';
+import * as os from 'os';
+import * as querystring from 'querystring';
+import { getBuffer } from './exportProvider';
+let sprintf = require("sprintf-js").sprintf;
```

Comment:
Can this be an import instead of require?  Just for consistency, unless there's some reason to have it be different.

Reply:
Resolved